isis/src/base/objs/RollingShutterCameraDetectorMap/RollingShutterCameraDetectorMap.cpp

```diff
@@ -148,8 +148,8 @@ namespace Isis {
     p_parentSample   = (p_detectorSample - p_ss) / p_detectorSampleSumming + 1.0;
     p_parentLine     = (p_detectorLine   - p_sl) / p_detectorLineSumming   + 1.0;
 
-    while((abs(sample - currentSample) < 1e-7) &&
-          (abs(line - currentLine) < 1e-7)) {
+    while((qFabs(sample - currentSample) < 1e-7) &&
+          (qFabs(line - currentLine) < 1e-7)) {
```

Comment:
For some reason, std::abs was rounding these very small numbers down to zero on Centos (almost as if it rounded using floats instead of actual doubles?). Replacing it with Qt's implementation seems to fix it.